unstructured/ingest/cli/cmds/weaviate.py

```diff
@@ -0,0 +1,63 @@
+import typing as t
+from dataclasses import dataclass
+
+import click
+
+from unstructured.ingest.cli.interfaces import CliConfig, Dict
+
+CMD_NAME = "weaviate"
+
+
+@dataclass
+class WeaviateCliConfig(CliConfig):
+    host_url: str
+    class_name: str
+    auth_keys: t.Optional[t.Dict[str, str]] = None
+
+    @staticmethod
+    def get_cli_options() -> t.List[click.Option]:
+        options = [
+            click.Option(
+                ["--host-url"],
+                required=True,
+                help="Weaviate instance url",
+            ),
+            click.Option(
+                ["--class-name"],
+                default=None,
+                type=str,
+                help="Target class collection name",
```

Comment:
We can add an additional sentence to explain what this is, and give 1-2 example (value[s]) in the help message